src/ui/components/WalletSourceIcon/WalletSourceIcon.tsx

```diff
@@ -25,5 +36,19 @@ export function WalletSourceIcon({
     <LedgerIcon
```

Comment:
Let's add `cutoutStroke` to other icons

Reply:
Maybe I misunderstood the design but it seemed to me that only "watch" one needs it 🤔

Reply:
What about Ledger?

Reply:
It follows the design currently, as I understand